.travis.yml

```diff
@@ -11,9 +11,12 @@ env:
   - secure: OHty0sCWbCAsxw8vLWmQLe2uBu8aR7zwKXh4aO8XipkNqIQSyVF3Q/iNC2DOclOmomT/C2wp2XWLysazJ5oyYpkooTe0wh9Bl2I3J6JqF7aynLzmk8eT0BoP/MZwX0lbCI11Z2JVW349xipku/vBVoucN40P6j5FfgZjTJYm5JE=
   - secure: SVw7sxOncYt1Tp9QoPLmKSw2IxpF1u3UyfmxIG3hqjkCVrNjYTehBfnS/pxoGVugrq8q9PQ9uJPZlmOp5IKIlJhyZ6kdpq2LjRxjgAPC935qOEgsu4OM1olE0G1oqdyDBv/tePmJFmr0ViUfCIIv464g+cYd81kFzcwaeXgKjOM=
   - DATABASE_URL=postgres://postgres@localhost:5432/travis_ci_test
+  - PATH=$PWD/bin:$PATH
 node_js:
 - '0.10'
 before_script:
+- curl -s http://www.princexml.com/download/prince-9.0r5-linux-amd64-static.tar.gz | tar xzf -
+- echo $PWD | ./prince-9.0r5-linux-amd64-static/install.sh; rm -rf ./prince-9.0r5-linux-amd64-static
```

Comment:
We should think about caching this so that we don't have to download for each build.